README.md

```diff
@@ -1,5 +1,10 @@
 # DevOps Apprenticeship: Project Exercise
 
+## This is for scripting for Automating
+ 
+The setup.sh file is script will error if there is an issue, setup Python, Poetry Install, Setup the dependencies. All you need is to copy TRELLO Credentials to .env for running the To Do App
```

Comment:
You might want to mention that this assumes python is already installed on the machine.